frontend/src/ArtistSearch/ArtistSearch.tsx

```diff
@@ -57,6 +61,12 @@ export default function ArtistSearchBar(props: IArtistSearchBarProps) {
     }
   }, [props.defaultArtist, artists]);
 
+  /**
+   * Enable editing for admin users
+   */
+  const userContext = useContext(Context)
+  useEffect(() => setIsAdmin(!!(userContext as any).user), [userContext]);
```

Comment:
what does !! do? just curious

Reply:
the JavaScript "[double bang](https://medium.com/@edplatomail/js-double-bang-or-the-not-operator-part-40e55d089bf0)" evaluates an object's existence to a boolean